.circleci/config.yml

```diff
@@ -580,67 +596,57 @@ workflows:
       - test-contracts:
           name: "test-main"
           requires: [build-testable]
-          workspace: "@synthetixio/main"
+          dir: "./protocol/synthetix"
           parallelism: 2
           batch-size: 8
 
       - test-contracts:
           name: "test-oracle-manager"
           requires: [build-testable]
-          workspace: "@synthetixio/oracle-manager"
+          dir: "./protocol/oracle-manager"
           parallelism: 1
           batch-size: 5
 
-      #- test-contracts: TODO: enable when we have some tests
-      #    name: "test-governance"
-      #    requires: [build-testable]
-      #    workspace: "@synthetixio/governance"
-
       - test-contracts:
           name: "test-spot-market"
           requires: [build-testable]
-          workspace: "@synthetixio/spot-market"
+          dir: "./markets/spot-market"
           parallelism: 2
           batch-size: 3
 
       - test-contracts:
           name: "test-perps-market"
           requires: [build-testable]
-          workspace: "@synthetixio/perps-market"
+          dir: "./markets/perps-market"
           parallelism: 8
           batch-size: 1
 
       - test-contracts:
           name: "test-bfp-market"
           requires: [build-testable]
-          workspace: "@synthetixio/bfp-market"
+          dir: "./markets/bfp-market"
           parallelism: 6
           batch-size: 1
 
       - test-contracts:
           name: "test-core-modules"
-          workspace: "@synthetixio/core-modules"
+          dir: "./utils/core-modules"
           parallelism: 1
           batch-size: 5
 
       - test-contracts:
           name: "test-core-contracts"
           requires: [build-testable]
-          workspace: "@synthetixio/core-contracts"
+          dir: "./utils/core-contracts"
           parallelism: 2
           batch-size: 5
 
       - test-contracts:
           name: "test-core-utils"
-          workspace: "@synthetixio/core-utils"
+          dir: "./utils/core-utils"
           parallelism: 2
           batch-size: 5
 
-      #- test-contracts: TODO: enable when tests are fixed
-      #    name: "test-legacy-market"
-      #    requires: [build-testable]
-      #    workspace: "@synthetixio/legacy-market"
-
```

Comment:
Maybe keep this? Or is legacy market not a thing anymore?

Reply:
It has some old config anyway. We can copy paste any other block from above and update for legacy market